### PR TITLE
Update code to use the "docker compose" syntax vice "docker-compose"

### LINKED ISSUE #54: Switch "docker-compose" to "docker compose".
# 💡 Summary #

The `docker` command is now capable of doing the work of `docker-compose`.    This message is now emitted by `docker-compose`:

```console
❱ docker-compose up
Docker Compose is now in the Docker CLI, try `docker compose up`
```

## Motivation and context ##

Eventually this will be the way™. 

## Implementation notes ##

https://docs.docker.com/compose/cli-command/

I've noticed that the new command is more pedantic about the `docker-compose.yml` files. 
Specifically, it does not permit quoted integers for port numbers.

## Acceptance criteria ##

How do we know when this work is done?

- [ ] all `docker-compose` references converted to `docker compose`
- [ ] all included `docker-compose.yml` files work with new command

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request updates the code to use the `docker compose` syntax instead of the `docker-compose` syntax.

## 💭 Motivation and context ##

The `docker compose` syntax is the preferred (and only correct) syntax after the changes in cisagov/ansible-role-docker#60.

## 🧪 Testing ##

Automated testing passes.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.
- [ ] All new and existing tests pass.

### CODE REVIEW

**Comment by jsf9k** [comment]
Note that the GitHub Actions should pass once #79 is merged.

**Review by dav3r — APPROVED** [review]
👍

File: docker-compose.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 version: "3.7"
 
-# This docker-compose file is used to build and test the container
+# This docker compose file is used to build and test the container
```
Comment (mcdonnnj) [inline docker-compose.yml]:
```suggestion
# This Docker composition file is used to build and test the container
```

**Review by mcdonnnj — APPROVED** [review]
Just one small suggestion. Thanks for getting us updated!

**Pull request merged**